bitnami/mongodb-sharded/README.md

```diff
@@ -288,11 +288,11 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
 
 ```bash
 $ helm install my-release \
-  --set mongodbRootPassword=secretpassword,mongodbUsername=my-user,mongodbPassword=my-password,mongodbDatabase=my-database \
+  --set shards=4,configsvr.replicas=3,shardsvr.dataNode.replicas=2 \
```

Comment:
Those parameters do not exist in the value.yaml, therefore they cannot be set.

Reply:
I'm confused, because I can find those parameters.

shards => https://github.com/bitnami/charts/blob/master/bitnami/mongodb-sharded/values.yaml#L75
configsvr.replicas => https://github.com/bitnami/charts/blob/master/bitnami/mongodb-sharded/values.yaml#L334
shardsvr.dataNode.replicas => https://github.com/bitnami/charts/blob/master/bitnami/mongodb-sharded/values.yaml#L86